tests/checkers/rstcheck.py

```diff
@@ -1,63 +1,34 @@
-"""Sanity test using rstcheck and sphinx."""
-from __future__ import annotations
-
-import re
-import subprocess
+from rstcheck_core.runner import RstcheckMainRunner
+from rstcheck_core.config import RstcheckConfig
```

Comment:
So are you using rstcheck or rstcheck-core?

Reply:
Yes,  I am using rstcheck_core.